website/content/docs/auth/azure.mdx

```diff
@@ -234,6 +234,63 @@ tool.
        https://127.0.0.1:8200/v1/auth/azure/role/dev-role
    ```
 
+## Root credential rotation
+
+If the mount is configured with credentials directly, the credential's key may be
+rotated to a Vault-generated value that is not accessible by the operator.
+This will ensure that only Vault is able to access the "root" user that Vault uses to
+manipulate dynamic & static credentials.
```

Comment:
```suggestion
Mounts can rotate root credential keys configured directly within the mount.
Rotating to a Vault-generated key makes the key value inaccessible to the
operator and ensures only Vault can operate as a root user to manipulate dynamic
and static credentials.
```
Style correction: write in active voice, avoid possessives

Reply:
applied to auth/azure and secrets/azure